secant/Stubs/MockServices.swift

```diff
@@ -65,7 +57,69 @@ class MockMnemonicPhraseHandling: MnemonicSeedPhraseHandling {
     func isValid(mnemonic: String) throws {}
 }
 
+class KeysPresentStub: KeyStoring {
+    init(returnBlock: @escaping () throws -> Bool) {
+        self.returnBlock = returnBlock
+    }
+    var returnBlock: () throws -> Bool
```

Comment:
Do you think we can reorganize this class to follow the Type structure that we put in the StyleGuide?

Reply:
I saw this was marked as resolved but I do think even stub files should follow our normal file structure guidelines.  Mixing the organization of inits, functions and properties only makes the code harder to reason about.  I see the point of having some local reasoning with variables near their functions, but overall the same standard file structure with well named variables and functions will be the easiest thing for someone else to read and understand.